### DISCUSSION TITLE
How to modify the loaded model package data with pandas and run_modelflow??

### DISCUSSION THREAD

**Discussion by ArtCoff** [modflowpy/flopy#2245] [Q&A] [answered]
I loaded a model of an actual venue using the modflow.load method and read the data from its packages.
package.stress_period_data.df method is really readable for me.
how do I modify its data via pandas and run mf.run_model with the modified data?
For example, I want to use pandas to modify the flux data inside the WEL package in the image and re-run the model to output the data.
![image](https://github.com/modflowpy/flopy/assets/57992286/a4abeff5-19f9-41be-8cd2-e039c6198ac8)

**Comment by p-ortega** [comment, marked as answer]
hey @GroundwaterSimulationOptimization 

Here is something to get you started

If you've modified the wel spd as needed into a new variable `new_spd`, then adding it to the `wel` package would be something like this:

```
wel.stress_period_data = new_spd # beware this is overwriting the attribute but YOLO
wel.write_file(f ='cool_name.wel')
```

it should work with `new_spd` being a df or dictionary

if you save the wel pack with a different name don't forget to update the `nam` file

Cheers
